clients/drcachesim/common/options.cpp

```diff
@@ -145,37 +145,41 @@ droption_t<unsigned int> op_num_cores(DROPTION_SCOPE_FRONTEND, "cores", 4,
 droption_t<unsigned int> op_line_size(
     DROPTION_SCOPE_FRONTEND, "line_size", 64, "Cache line size",
     "Specifies the cache line size, which is assumed to be identical for L1 and L2 "
-    "caches.  Must be a power of 2.");
+    "caches.  Must be at least 4 and a power of 2.");
 
-droption_t<bytesize_t> op_L1I_size(
-    DROPTION_SCOPE_FRONTEND, "L1I_size", 32 * 1024U, "Instruction cache total size",
-    "Specifies the total size of each L1 instruction cache.  Must be a power of 2 "
-    "and a multiple of -line_size.");
+droption_t<bytesize_t>
+    op_L1I_size(DROPTION_SCOPE_FRONTEND, "L1I_size", 32 * 1024U,
+                "Instruction cache total size",
+                "Specifies the total size of each L1 instruction cache."
+                " L1I_size/L1I_assoc must be a power of 2 and a multiple of -line_size.");
```

Comment:
The leading dash bothers me as well.  How about if I remove the dash from all of them, since really this text is disussing cache parameters which are sometimes but not always set via command-line options.

Reply:
I think there should be something indicating a literal: maybe quotes of some sort if there's no dash.  E.g. 'line_size'.

Reply:
You don't think the underscore is indication enough?

Reply:
Do they all have underscores?  If so, maybe that is enough.

Reply:
I believe all the cache parameters except "coherence" contain underscores, and the coherence option text never uses that word in a way that is ambiguous.  So I don't think there is any ambiguity between literals and just words.

Reply:
OK, sure.